internal/server/v1/observations/golden/bulk_point_linked/all_Country.json

```diff
@@ -6,11 +6,6 @@
         {
           "entity": "country/ABW",
           "points_by_facet": [
-            {
-              "date": "2021",
```

Comment:
Seems like we've lost 2021 data here. Is this expected? @n-h-diaz @ajaits @pradh

Reply:
Please follow this thread: https://github.com/datacommonsorg/mixer/pull/1310#discussion_r1442116083